### DISCUSSION TITLE
api to scroll to bottom or top?

### DISCUSSION THREAD

**Discussion by bsc0tt** [nolimits4web/swiper#4562] [Q&A] [answered]
Using a scroll container, Is there a method to in the api to go to the bottom or top of the page?

**Comment by bsc0tt** [comment]
for forcing to the bottom I did this: `swiper.wrapperEl.style.transform=translate3d(0px, -${swiper.snapGrid[1]}px, 0px)`

**Comment by ajaypillay** [comment, marked as answer]
Is there any reason you couldn't do `swiper.slideTo(0)` and `swiper.slideTo(array.length - 1)` or `swiper.slideTo(lastIndex)`. If you keep track of the number of slides within your Swiper then this could be done.

**Reply by nolimits4web** [reply]
Yes, also `swiper.setProgress(0)` (beginning) and `swiper.setProgress(1)` should do the trick

**Reply by bsc0tt** [reply]
Yes, thanks!  slideTo() and  swiper.setProgress() both work great and did not think to use them because I only had one slide(the scroll container).
